.github/CODEOWNERS

```diff
@@ -528,7 +528,7 @@
 
 # > Service: Network
 # ServiceLabel: %Network %Service Attention
-#/<NotInRepo>/            @aznetsuppgithub
+sdk/network/           @aznetsuppgithub
```

Comment:
This folder only contains the resource manager for network... That might be the right thing -- but not 100% sure.